src/components/TryExtensionLinkDialog.tsx

```diff
@@ -122,7 +122,11 @@ const TryExtensionBox: FunctionComponent<{
             again.
           </div>
         )}
-        <button type="button" onClick={clickHandler}>
+        <button
+          data-testid="try-extension-button"
```

Comment:
oof, testIDs are a brittle way to do testing; is there not a more robust mechanism to find the only button here?